libs/types/index.ts

```diff
@@ -54,7 +54,6 @@ export type { Duration } from './models/Duration';
 export type { EnrollmentConfig } from './models/EnrollmentConfig';
 export type { EnrollmentRequest } from './models/EnrollmentRequest';
 export type { EnrollmentRequestApproval } from './models/EnrollmentRequestApproval';
-export type { EnrollmentRequestApprovalStatus } from './models/EnrollmentRequestApprovalStatus';
```

Comment:
can you run the type generation again ? We just merged PR which adds `EnrollmentRequestApprovalStatus`

Reply:
Rolled back, will update types separately if needed.